docs/develop-resources/scenarios/new-field.md

```diff
@@ -11,23 +11,22 @@ REPO_ROOT="$(git rev-parse --show-toplevel)"
 cd $REPO_ROOT/dev/tools/controllerbuilder
 
 go run . update-types \
-    --parent-message-full-name "google.monitoring.dashboard.v1.Dashboard" \
-    --new-field "row_layout" \
+    --parent-message "google.monitoring.dashboard.v1.Dashboard" \
+    --field-to-insert "row_layout" \
     --api-dir ${REPO_ROOT}/apis/monitoring/v1beta1
 ```
 
-* `--parent-message-full-name`
+* `--parent-message`
 
 Fully qualified name of the proto message holding the new field.
 
-* `--new-field`
-
-Name of the new field.
+* `--field-to-insert`
```

Comment:
May I ask why we make this change? I think `new-field` and `field-to-insert` mean the same thing but the latter is a less typical name.

Reply:
We could revert back to `--new-field`, or call it `--insert-field`. In my mind, I was thinking maybe in the future we also want to have flags for removing field, update a field (new comment), etc.